onnx_mxnet/backend.py

```diff
@@ -49,7 +49,8 @@ def run_node(cls, node, inputs, device='CPU'):
         data_names = [i for i in node.input]
         data_shapes = []
         reduce_op_types = set(['ReduceMin', 'ReduceMax', 'ReduceMean',
```

Comment:
Looks good to me.
Change this variable name to something more relevant i.e dimChange_op_types or something other than "reduce"

Reply:
I will do the variable name change in one of the subsequent PRs.